android/java/org/chromium/chrome/browser/app/domain/CryptoModel.java

```diff
@@ -197,24 +194,12 @@ public void getDecryptMessageRequest(Callback1<DecryptRequest> onResult) {
 
     public void refreshTransactions() {
         synchronized (mLock) {
-            if (mKeyringService == null || mPendingTxHelper == null
-                    || mBraveWalletService == null) {
+            if (mKeyringService == null || mPendingTxHelper == null) {
                 return;
             }
-            List<Integer> coins = new ArrayList<>();
-            for (CryptoAccountTypeInfo cryptoAccountTypeInfo :
-                    mSharedData.getSupportedCryptoAccountTypes()) {
-                coins.add(cryptoAccountTypeInfo.getCoinType());
-            }
 
-            mKeyringService.getAllAccounts(allAccounts -> {
-                new SelectedAccountResponsesCollector(mKeyringService, mJsonRpcService, coins,
-                        Arrays.asList(allAccounts.accounts))
-                        .getAccounts(mOrigin, defaultAccountPerCoin -> {
-                            mPendingTxHelper.setAccountInfos(
-                                    new ArrayList<>(defaultAccountPerCoin));
-                        });
-            });
+            mKeyringService.getAllAccounts(
+                    allAccounts -> { mPendingTxHelper.setAccountInfos(allAccounts.accounts); });
         }
```

Comment:
We don't have per-coin accounts now. And we can fetch transaction for all accounts.